frontend/alert/src/App.js

```diff
@@ -3,12 +3,9 @@ import "./App.css";
 import styled from "styled-components";
 import PropTypes from "prop-types";
 import Logo from "./assets/PCA_logo.svg";
-
+import ManageAlertWrapper from "./components/managealert/index";
```

Comment:
```suggestion
import ManageAlertWrapper from "./components/managealert";
```
index.js files are the default for the directory, so no need to add `index`! I'd also recommend renaming the directory to `ManageAlerts` to match casing for components.

Reply:
oh wow! i did not know they default to index.js - learned something new :)

Reply:
We actually get a linter error for having a directory name with uppercases - seems like it's due to the fact that some file systems are case insensitive, and that screws with the code base.

Reply:
jk, this seems to be there since the beginning before I even moved directories. It seems to have not come from ESLint, but my editor. I'll investigate

Reply:
Never mind - the directory was the issue, after resetting to head my projectile cache still used the upper case directory which is why my linter was complaining. I'm gonna use lower case directory names then.